coral-spark/src/main/java/com/linkedin/coral/spark/spark2rel/Spark2CoralOperatorTransformerMap.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.spark.spark2rel;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.calcite.sql.type.ReturnTypes;
+
+import static com.linkedin.coral.spark.spark2rel.Spark2CoralOperatorTransformerMapUtils.createOperator;
+import static com.linkedin.coral.spark.spark2rel.Spark2CoralOperatorTransformerMapUtils.createTransformerMapEntry;
+import static com.linkedin.coral.spark.spark2rel.Spark2CoralOperatorTransformerMapUtils.getKey;
+import static org.apache.calcite.sql.type.OperandTypes.STRING_STRING;
+
+
+public class Spark2CoralOperatorTransformerMap {
+  private Spark2CoralOperatorTransformerMap() {
+  }
+
+  public static final Map<String, OperatorTransformer> TRANSFORMER_MAP = new HashMap<>();
+
+  static {
+    // TODO: keep adding Spark-Specific functions as needed
+
+    createTransformerMapEntry(TRANSFORMER_MAP, createOperator("strpos", ReturnTypes.INTEGER, STRING_STRING), 2,
+        "instr");
```

Comment:
This sounds wrong since Spark already uses `instr`?

Reply:
Removed this line.